packages/components/src/components/vertical-tab/vertical-tab-content-container.jsx

```diff
@@ -23,24 +46,18 @@ const Content = ({ is_routed, items, selected }) => {
                             <Route
                                 key={idx}
                                 path={path}
-                                render={() => <Component component_icon={icon} setSideNotes={setSideNotes} />}
+                                render={() => <Component component_icon={icon} setSideNotes={addToNotesQueue} />}
```

Comment:
```suggestion
                                render={() => <Component component_icon={icon} addToNotesQueue={addToNotesQueue} />}
```

Reply:
imo, the children expect that they are setting side notes, where are the parent control how to set side notes 🙏

Reply:
By this comment I mean if we can use same name for key and values here? Why two distinct names for a unit of work?

Reply:
got it, its a props, not exactly key, value. imo, component is the children, it only expecting to set a side notes, it doesn't know what is addToNotesQueue. only parent has that function and controls how to set notes from there. same like the sibling props component_icon, it accepts icon, not component_icon